src/Documentation/sidebar.json

```diff
@@ -8,19 +8,19 @@
       "initialize",
       "configure",
       "add-files",
-      "share-data",
+      "store-data",
       "retrieve-data",
+      {
+        "label": "Import Data",
```

Comment:
you don't need to specify a label if it's the same as title in .md file

Reply:
Ah true. This used to be different than it's title, that's why it was expanded here. Fixed.